pkg/apis/serving/v1beta1/component.go

```diff
@@ -236,7 +236,7 @@ const (
 
 type MetricSource struct {
 	// MetricsBackend defines the scaling metric type watched by autoscaler
-	// possible values are prometheus, graphite.
+	// possible values are prometheus, graphite, opentelemetry.
```

Comment:
We should also consider adding the PodMetricsSource type?

Reply:
Now you can remove opentelemetry, ExternalMetricSource means autoscaler get the metrics from external source like prometheus, graphite. For PodMetricsSource autoscaler gets directly from Pod.